app/dashboard/models.py

```diff
@@ -2704,6 +2742,8 @@ def to_dict(self, activities=True, leaderboards=True, network=None, tips=True):
         works_with_funded = self.get_who_works_with(work_type='funded', bounties=funded_bounties)
         works_with_collected = self.get_who_works_with(work_type='collected', bounties=fulfilled_bounties)
 
+        sum_all_funded_tokens = self.get_all_tokens_sum(sum_type='funded', bounties=funded_bounties)
```

Comment:
Why is the network not being passed in here?

Reply:
Because this function has defined default network.

Reply:
@rafalkowalski yes but if you run this locally on rinkeby it will give you the wrong information, right?

Reply:
@danlipert Yes, Do we have any function/method where i can verify current network?

Reply:
its passed into this function as the `network` kwarg

Reply:
Thanks, @danlipert done